defi/src/governance/icp.ts

```diff
@@ -1,82 +1,87 @@
 import axios from 'axios'
 import { GovCache, Proposal } from './types';
-// import ic from 'ic0';
+import ic from 'ic0';
 import { PromisePool } from "@supercharge/promise-pool";
 import { updateStats } from './utils';
 import { setCompound, getCompound } from './cache';
+import * as fs from 'fs';
```

Comment:
not sure where this is used, can be removed?

Reply:
aggreed, removed!